### PR TITLE
Bug 1885173: Add VM next run configuration test

### PR DESCRIPTION
**Pull request by irosenzw**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { $, $$ } from 'protractor';
+
+export const isPCAlertPresent = async (): Promise<boolean> =>
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts:3]: there is a quite overlap with pendingChangesInfoAlert and pendingChangesWarningAlert

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  });
+
+  it('Change Boot-Order while VM is running.', async () => {
+    await vm.navigateToDetail();
+    await vm.modalEditBootOrder();
+    expect(await isPCInfoAlertPresent()).toBeTruthy();
+    await click(bootOrderView.deleteDeviceButton(1));
+    expect(await isPCAlertPresent()).toBeTruthy();
+    await click(saveButton);
+    expect(await isPCAlertPresent()).toBeTruthy();
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts:108]: this line shouldn't be here

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  }, VM_BOOTUP_TIMEOUT_SECS);
+
+  afterEach(async () => {
+    await vm.detailViewAction(VM_ACTION.Stop);
+  }, VM_BOOTUP_TIMEOUT_SECS);
+
+  it('Change Flavor from tiny to custom while VM is running.', async () => {
+    await vm.navigateToDetail();
+    await vm.modalEditFlavor();
+    expect(await getSelectedOptionText(editFlavorView.flavorDropdown)).toEqual('Tiny');
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts:64]: we have enum for these (please fix other places as well)

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  const vm = new VirtualMachine(testVM.metadata);
+  const configmapName = 'configmap-mock';
+  const configMap = getConfigMap(testName, configmapName);
+
+  beforeAll(async () => {
+    createResources([configMap, multusNAD, testVM]);
+  });
+
+  afterAll(() => {
+    deleteResources([configMap, testVM]);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts:48]: shouldn't we deleted multusNAD here as well?

**Review by suomiy — CHANGES_REQUESTED** [review]

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts (hunk 2 above)
Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.next.run.configuration.scenario.ts:108]: why not ?

File: frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts (hunk 1 above)
Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts:3]: what is you point ? each has unique class that I'm using to identify it.

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/views/vm.next.run.configuration.view.ts:3]: that they were using the same class, but the constants file is gone now - so it is fine

**Pull request merged**